### DISCUSSION TITLE
blog/pengalaman-menggunakan-prisma-orm

### DISCUSSION THREAD

**Discussion by giscus** [hendraaagil/website#47] [Comments]
# blog/pengalaman-menggunakan-prisma-orm

Pengalaman saya ketika menggunakan Prisma ORM untuk pertama kalinya.

https://hendraaagil.dev/blog/pengalaman-menggunakan-prisma-orm

**Comment by sidiqalfi** [comment]
keren nih ke index google, iseng search "apa itu prisma orm" malah muncul web ini, semangat bang👏

**Reply by hendraaagil** [reply]
mantap bang :+1:
